### ISSUE TITLE
Missing apostrophe in string display of Queryable expression

### ISSUE THREAD

**Issue by FreHu** [dotnet/runtime#80069]
### Description

I copy-pasted an expression from the Visual Studio debug view into a vscode window. Because my rainbow braces were messed up, I noticed that there is a missing apostrophe in the output.

### Reproduction Steps

You can reproduce with this simple program
```c#
var list = new List<string>().AsQueryable();
var expr = list
    .Where(x => x.Contains("test") && x.Contains("test2"));

Console.ReadLine();
```
Now place a breakpoint and get the string from the debug view
![image](https://user-images.githubusercontent.com/5097067/210075945-5c0301d3-f7cf-4801-9ea3-f3fb5602320f.png)
![image](https://user-images.githubusercontent.com/5097067/210076340-c505b857-ac56-47f0-8c4e-0435de9811aa.png)

```c#
.Call System.Linq.Queryable.Where(
    .Constant<System.Linq.EnumerableQuery`1[System.String]>(System.Collections.Generic.List`1[System.String]),
    '(.Lambda #Lambda1<System.Func`2[System.String,System.Boolean]>))

.Lambda #Lambda1<System.Func`2[System.String,System.Boolean]>(System.String $x) {
    .Call $x.Contains("test") && .Call $x.Contains("test2")
}
```

### Expected behavior

The correct output should look like this
```c#
.Call System.Linq.Queryable.Where(
    .Constant<System.Linq.EnumerableQuery`1[System.String]>(System.Collections.Generic.List`1[System.String]),
    '(.Lambda #Lambda1<System.Func`2[System.String,System.Boolean]>)')

.Lambda #Lambda1<System.Func`2[System.String,System.Boolean]>(System.String $x) {
    .Call $x.Contains("test") && .Call $x.Contains("test2")
}
```
![image](https://user-images.githubusercontent.com/5097067/210076418-8bdc4441-1b77-4794-b5b1-e55a9e10a7f2.png)

### Actual behavior

The apostrophe is not properly terminated
![image](https://user-images.githubusercontent.com/5097067/210076037-c7b7b595-2fdb-4853-bf8a-f2ba217d5307.png)

### Known Workarounds

I suppose you can fill in the apostrophe yourself.

### Configuration

![image](https://user-images.githubusercontent.com/5097067/210076790-e3215b3e-e358-4dd9-8300-666c4d245265.png)
![image](https://user-images.githubusercontent.com/5097067/210076843-e9a70f86-09c7-47e2-988e-d0aa0db21c31.png)

### Other information

I believe this is not an editor issue, but the editor is calling some method which returns the string without an apostrophe.

**Comment by svick** [comment]
That apostrophe is representing an `UnaryExpression` with kind of `Quote`:

https://github.com/dotnet/runtime/blob/5e45fb1d67e868b0ca67f819261cf5d8ff8652c5/src/libraries/System.Linq.Expressions/src/System/Linq/Expressions/DebugViewWriter.cs#L906-L908

It's not meant to be a string, or anything like that, so I think having a single apostrophe is intentional.

And since the debug view does not follow any well-defined syntax, I don't think there's anything wrong with the current approach.

**Comment by FreHu** [comment]
I see. I did not know that `Expression.Quote` was a thing.

_The quote operator is an operator which induces closure semantics on its operand._
https://stackoverflow.com/questions/3716492/what-does-expression-quote-do-that-expression-constant-can-t-already-do

So what I'm really looking at in 
```
'(.Lambda #Lambda1<System.Func`2[System.String,System.Boolean]>)
```` 
is  `Expression.Quote(...lambda...)` and the parentheses enclose what is "quoted". It's just a useful coincidence that C# highlighting assigns mostly valid coloring to an expression debug output. For this purpose, the choice of using `'` is a bit unfortunate, but there's nothing incorrect here. 

Closing.
